platform/broadcom/sonic-platform-modules-inventec/d6356/utils/inventec_d6356_util.py

```diff
@@ -141,7 +141,7 @@ def system_install():
     status, output = exec_cmd("rmmod lpc_ich ", 1)
 
     #insert extra module
-    status, output = exec_cmd("insmod /lib/modules/4.9.0-11-2-amd64/kernel/drivers/gpio/gpio-ich.ko gpiobase=0",1)
+    status, output = exec_cmd("insmod /usr/lib/modules/4.19.0-9-2-amd64/kernel/drivers/gpio/gpio-ich.ko gpiobase=0",1)
```

Comment:
this is easily broken. we should not hardcode the driver path into the installer.

Reply:
This part is modified after previous feedback, please help to review, thank you!